src/components/layout/CompanyClaim/CompanyClaim.module.scss

```diff
@@ -69,4 +69,31 @@
       }
     }
   }
+
+  @media all and (max-width: 500px) {
+    img {
+      display: flex;
+      align-items: left;
+      margin: -80px 20px;
+    }
+    .cart {
+      .cartBox {
```

Comment:
If we are using the SCSS - there is a simplest way to create a child element - try something like this:
```scss
.cart {
    &Box {
    }
}
```
this is the same, but more powerful :)